doc/.ChangeLog_template

```diff
@@ -48,45 +52,42 @@ Changes made to namelist defaults (e.g., changed parameter values):
 
 Changes to the datasets (e.g., parameter, surface or initial files):
 
-Substantial timing or memory changes: [For timing changes, can check PFS test(s) in the test suite]
+Substantial timing or memory changes:
+[For timing changes, you should at least check the PFS test in the test
+ suite, whether or not you expect timing changes: ensure that its run
+ time has not increased significantly relative to the baseline.]
+
 
-Notes of particular relevance for developers: (including Code reviews and testing)
+Notes of particular relevance for developers:
 ---------------------------------------------
 NOTE: Be sure to review the steps in README.CHECKLIST.master_tags as well as the coding style in the Developers Guide
+[Remove any lines that don't apply. Remove entire section if nothing applies.]
 
 Caveats for developers (e.g., code that is duplicated that requires double maintenance):
 
 Changes to tests or testing:
 
-CTSM testing:
-
-[... Remove before making master tag.  Available test levels:
-
-    a) regular (must be run before handing off a tag to SEs and must be run
-     before committing a tag)
-    b) build_namelist (if namelists and/or build_system changed))
-    c) tools (only if tools are modified and no CTSM source is modified)
-    d) short (for use during development and in rare cases where only a small
-          change with known behavior is added ... eg. a minor bug fix)
-    e) doc (no source testing required)
 
-... ]
+Testing summary
+---------------
+[Remove any lines that don't apply.]
 
- [PASS means all tests PASS and OK means tests PASS other than expected fails.]
+ [PASS means all tests PASS; OK means tests PASS other than
+  expected fails, including expected baseline failures.]
 
-  build-namelist tests:
+  build-namelist tests (if CLMBuildNamelist.pm has changed):
 
     cheyenne - 
 
-  tools-tests (test/tools):
+  tools-tests (test/tools) (if tools have been changed):
 
     cheyenne - 
 
   PTCLM testing (tools/shared/PTCLM/test):
 
     cheyenne - 
 
-  python testing (see instructions in python/README.md; document testing done):
+  python testing (if python code has changed; see instructions in python/README.md; document testing done):
```

Comment:
Just a general note. Since these unit tests are quick, I always run them. It doesn't really hurt. And technically changes on the system could show up between tags that cause the test to fail.

Maybe this would be a good candidate to add automated testing for when a PR is submitted?

Reply:
> Maybe this would be a good candidate to add automated testing for when a PR is submitted?

Yeah, this would be an easier thing to add automated testing for; on the other hand, it wouldn't have a ton of value right now because the python code changes in so few PRs currently... but maybe that will change moving forward as we move more of our tooling (including build-namelist) to python.

Reply:
Right part of what I was thinking was just that it would likely be easy to add in. And yes I expect that with the tool chain updates it will become more useful. Doing simple automated testing when we can, enables us to eventually figure out something that would be more complex, but more useful.

Reply:
Agreed.